config/mcp_config.py

```
@@ -0,0 +1,75 @@
+            },
+            "tools": {
+                "enabled": ["code_analyzer", "protocol_validator", "self_corrector"],
+                "timeout": 30
+            }
+        }
+        
+    def get_config(self) -> Dict[str, Any]:
+        """Get current configuration"""
+        if self.config_path.exists():
+            try:
+                with open(self.config_path, 'r') as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)